charts/dapr/charts/dapr_sidecar_injector/templates/dapr_sidecar_injector_deployment.yaml

```diff
@@ -147,6 +147,8 @@ spec:
 {{- end }}
         - name: SIDECAR_RUN_AS_NON_ROOT
           value: "{{ .Values.sidecarRunAsNonRoot }}"
+        - name: DROP_CAPABILITIES
```

Comment:
Can you change this to `SIDECAR_DROP_CAPABILITIES` for consistency?

Reply:
Or maybe even `SIDECAR_DROP_ALL_CAPABILITIES`

Reply:
Good point! I updated the name for consistency